packages/patrol/example/bs_ios

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# bs_ios uploads app binaries for UI testing on BrowserStack.
+#
+# This version of the script works only with the V2 endpoint (XC Test Plans).
+# This scirpt assumes that Xcode Test Plan name is Patrol
```

Comment:
- typo
- this should be configurable, or even better: required to be passed as argument